nengo/builder/builder.py

```diff
@@ -132,15 +132,8 @@ def build(self, obj, *args, **kwargs):
             The object to build into this model.
         """
         built = self.builder.build(self, obj, *args, **kwargs)
-
-        def NoneFunction(other):
-            return
-
-        function_to_call = NoneFunction
         if self.build_callback is not None:
-            function_to_call = self.build_callback
-        if function_to_call is not NoneFunction:
-            function_to_call(obj)
+            self.build_callback(obj)  # pylint: disable=not-callable
```

Comment:
Does adding `and callable(self.build_callback)` silence the pylint warning? Would be more explicit.